benchmarks/nemo/nemolite2d/psykal/psyclone_scripts/serial_transform.py

```diff
@@ -0,0 +1,18 @@
+''' Python script intended to be passed to PSyclone's generate()
+function via the -s option. '''
```

Comment:
Please say what it does, e.g. "module-inlines all kernels into the PSy layer"

Reply:
Done